contractcourt/htlc_timeout_resolver.go

```diff
@@ -551,21 +532,15 @@ func (h *htlcTimeoutResolver) sweepSecondLevelTx() error {
 // resolveSecondLevelTxLegacy sends a second level timeout transaction to the
 // utxo nursery. This transaction uses the legacy SIGHASH_ALL flag.
 func (h *htlcTimeoutResolver) resolveSecondLevelTxLegacy() error {
-	log.Debugf("%T(%v): incubating htlc output", h,
-		h.htlcResolution.ClaimOutpoint)
+	h.log.Debug("incubating htlc output")
```

Comment:
"incubating" indicates it's legacy tho

Reply:
Nit: so feel free to ignore:  so maybe we switch incubating for legacy, I find incubating a bit confusing ?

Reply:
the context knowledge is if the input is sent to the incubator it must be legacy, and this will soon be [removed](https://github.com/lightningnetwork/lnd/pull/9276#discussion_r1853286957).